data/categories.txt

```diff
@@ -4775,7 +4778,7 @@ nl:3^Berghut|alpenhut|hotel
 fi:^Vuorimajoitus|hotelli
 fr:^Hôtel de montagne|hôtel|auberge de jeunesse|cabane alpine
 de:4^Berghütte|Almhütte|Berghotel|Hotel|Unterkunft
-hu:^Hegyi menedék|kabin|hotel|szálloda
+hu:^Menedékház (személyzettel)|alpesi menedékház|szálloda|ifjúsági szálló|hostel
```

Comment:
Why parentheses?

Reply:
You may have right: it is not indispensable so you can leave it. (In another application there are two types of huts, one is with staff and the other is without staff and they are indicated in parentheses. But for maps.me it is perfect also without parenteses.)